src/components/text/heading/style.ts

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Font weight, letter-spacing and line-height variates between different headers. Please add them based on the design language

Reply:
ok talked with @rossmoody and we decided to leave letter-spacing/line-height open atm. new spec available at https://app.goabstract.com/projects/505d2110-74ea-11e8-b35b-eb800813dd0e/branches/master/commits/1e57e5e86759ba5add339fc07d17baa4beae372e/files/C9D94365-DAF9-4758-B1F6-9310D9C8157A/layers/D4259C65-D83D-4B35-BEEA-B7168CC3CA11?mode=design

Reply:
this pr is updated with the above

Reply:
thank you